kolibri/plugins/management/assets/src/views/user-page/index.vue

```diff
@@ -182,18 +182,20 @@
       showCreateUserModal() {
         return this.modalShown === constants.Modals.CREATE_USER;
       },
-      manageUserOptions() {
-        return [
-          { label: this.$tr('editUser') },
-          { label: this.$tr('resetUserPassword') },
-          { label: this.$tr('deleteUser') },
-        ];
-      },
     },
     methods: {
       userMatchesRole(user) {
         return this.roleFilter === 'all' || user.kind === this.roleFilter;
       },
+      manageUserOptions(userId) {
```

Comment:
Can you DRY this up a bit? Make `options` something like this:

```javascript
const options = [
  /* first two */,
 { label: this.$tr('deleteUser'), disabled: userId === this.currentUserId }
];
return options;
```

Reply:
T R U E